README.md

```diff
@@ -95,9 +180,30 @@ make -j4
 ```sh
 ./build/bin/OpenApoc
 ```
+## How_to_setup_OpenApoc
 
-## Contact us
-
-If you're interested, please visit our [website](http://openapoc.org).
-We also have [forums](http://openapoc.org/forums/) - please pop by and introduce yourself!
-We also have an IRC channel on [Freenode](http://freenode.net) - [#openapoc](irc://irc.freenode.net/#openapoc).
+* This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be acquired from [Steam](http://store.steampowered.com/app/7660/) - this is _required_ to run)
+  - you need have all files in ISO file including MUSIC etc
+  - if it's in .iso format, rename it to "cd.iso"
+  - if it's not, copy all the contents into a folder and rename the folder to "cd.iso"
```

Comment:
We also support the .cue / .bin files (which are used, for example, in the gog.com version)

You rename the .cue file "cd.iso", put it in the data/ folder, then put the .bin file in the same folder (without changing it's name - so for example the gog.com file remails "XCOM.BIN")

That's in addition to a directory called "cd.iso" :)

Reply:
Change it